app/_components/pagelink/PageLink.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Badge,
+  Heading,
+  LinkBox,
+  LinkOverlay,
+  useColorMode,
+} from '@chakra-ui/react';
+import NextLink from 'next/link';
+
+interface Props {
+  url: string;
+  icon: React.ReactNode;
+  title: string;
+  description: string;
+  notImplemented?: boolean;
+}
+
+function PageLink({
```

Comment:
Too generic a name for a specific component. PageLink comes off as a component that handles some base config for how we want to handle links for the pages. I would suggest using something that reflects the component use. IMO for this use case, if the base chakra component handles most of the UX needed, we don't really need to make a new component for it. Also, lose the notImplemented, we can handle this by keeping things in our dev stack until they are ready to be deployed to production.